packages/node/src/module.ts

```diff
@@ -1,45 +1,63 @@
-import { basename, dirname } from '@sentry/utils';
+import { posix, sep } from 'path';
+
+const isWindowsPlatform = sep === '\\';
 
 /** normalizes Windows paths */
-function normalizePath(path: string): string {
```

Comment:
til, but a single call to replace is slower than a call to .test followed by .replace as replace will always allocate a new string.